src/Social/Models/MessageComments.php

```diff
@@ -132,4 +132,25 @@ public function getParentId(): int
     {
         return $this->parent_id == 0 ? $this->getId() : $this->parent_id;
     }
+
+    /**
+     * Check if comment is parent.
+     *
+     * @return bool
+     */
+    public function isParent() : bool
+    {
+        return $this->parent_id == 0;
+    }
+
+    /**
+     * Verify if this comment has message
+     *
+     * @param Messages $message
+     * @return boolean
+     */
+    public function hasMessage(Messages $message): bool
```

Comment:
Why do we need this ?. And, if this is really needed, why dont we check against the messages table to verify if the message really exists or not. @arfenis

Reply:
When you make a comment or a reply normally you do it from a message object/id, this validate if that comment/reply its indeed from that message. @rwhite27